nikola/data/themes/base-jinja/templates/math_helper.tmpl

```diff
@@ -0,0 +1,58 @@
+{% macro math_scripts() %}
```

Comment:
How about adding two macros (one for KaTeX, one for MathJax) which provide the version number for each of them?